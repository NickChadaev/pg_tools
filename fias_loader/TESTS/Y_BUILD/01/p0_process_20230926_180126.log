2023-09-26 18:01:26 home/n.chadaev@abrr.local
127.0.0.1 5432 unsi_test_01 postgres /home/n.chadaev@abrr.local/Y_BUILD/01/stage_gar_c_01.csv 223-09-25
--------------------------------------------------------
[18:01:26] ... Адыгея Респ, id_region: 2, Execute: /home/n.chadaev@abrr.local/Y_BUILD/01/stage_gar_c_01.csv, process_id:5456
[18:01:26] ... Start process
[18:01:26] ...  -- Version
[18:01:26] ...  -- Очистка от данных
[18:01:26] ...  -- Set UNLOGGED
[18:01:26] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_APARTMENT_TYPES_20230925_dec7f8ad-4238-4484-bf99-113a09c762f5.XML
[18:01:26] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_ADDR_OBJ_TYPES_20230925_4f714402-bdf0-4b03-8e2f-cda2704fb403.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_ROOM_TYPES_20230925_6fedd268-7191-415b-be0e-8a5a110f7099.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_OPERATION_TYPES_20230925_d65578ac-f625-4e50-ac1b-06ee9ac4de7a.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_PARAM_TYPES_20230925_72393625-9278-4105-b37d-79dc852810ae.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_HOUSE_TYPES_20230925_1bdc09e7-b8f0-40d5-80f3-cf003785714b.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_ADDHOUSE_TYPES_20230925_cf9caa45-fb3f-40d3-8ddf-770f536de4ed.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_OBJECT_LEVELS_20230925_8ba71762-ad23-4052-b0b8-a659eac50bbb.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_NORMATIVE_DOCS_TYPES_20230925_acf1cb3d-a66a-4e4b-b233-cc9c4032d98a.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_NORMATIVE_DOCS_KINDS_20230925_a8475f50-3b47-4725-9f46-b1bd99ea576c.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/01/AS_REESTR_OBJECTS_20230925_b4491030-6ef8-4a9b-94f1-5bcb1e4432e4.XML
[18:05:13] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/01/AS_ADDR_OBJ_20230925_02561b09-c3fd-448a-988a-867af269fda5.XML
[18:05:20] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/01/AS_HOUSES_20230925_3fd94ee2-a707-4e1c-82d1-b2c5f1203068.XML
[18:10:53] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/01/AS_STEADS_20230925_6d430d45-b3ed-46af-bc16-bcd1c0f9208d.XML
[18:11:29] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/01/AS_APARTMENTS_20230925_402c1c44-beea-4f1c-bfee-a0bcea5dd5a8.XML
[18:12:50] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/01/AS_ROOMS_20230925_a0331b46-0a20-4f5e-8d3d-30aa34933ada.XML
[18:12:51] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/01/AS_CARPLACES_20230925_af983ade-a52b-4a7b-8292-5ef2d2d90c4e.XML
[18:12:51] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/01/AS_ADM_HIERARCHY_20230925_4ca42f5e-8cd3-4db8-ba06-1d9cc60215bc.XML
[18:17:36] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/01/AS_MUN_HIERARCHY_20230925_1b113f45-37b3-44f0-b600-d7954eed4c66.XML
[18:21:40] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/01/AS_CHANGE_HISTORY_20230925_d5a479fb-3bb3-4194-8622-f7af5800c401.XML
[18:36:28] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/01/AS_ADDR_OBJ_DIVISION_20230925_7a902876-bdcf-4950-a35f-007ddc704b96.XML
[18:36:28] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/01/AS_ADDR_OBJ_PARAMS_20230925_eb10bfb2-2642-4365-8f1e-98a4b959dfc3.XML
[18:37:54] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/01/AS_STEADS_PARAMS_20230925_ba8061e3-eae2-4e6a-8288-b78148f1d44e.XML
[18:44:50] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/01/AS_HOUSES_PARAMS_20230925_b0368120-3230-4427-93de-54a7abaf89d7.XML
[19:14:04] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/01/AS_APARTMENTS_PARAMS_20230925_32d510ee-c59b-41e4-a1ae-0cb353d07c6d.XML
[19:18:12] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/01/AS_ROOMS_PARAMS_20230925_65319152-2e3e-4b96-84a3-8d261c7ed539.XML
[19:18:12] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/01/AS_CARPLACES_PARAMS_20230925_89aca207-4ff0-417c-bf0b-3f3d45c7b99e.XML
[19:18:13] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/01/AS_NORMATIVE_DOCS_20230925_cc628fd2-38b9-4eb6-b93a-753a78980a16.XML
[19:18:52] ...  -- Data finish
[19:18:52] ...  -- Set LOGGED
[19:19:06] ... Stop process
--------------------------------------------------------
