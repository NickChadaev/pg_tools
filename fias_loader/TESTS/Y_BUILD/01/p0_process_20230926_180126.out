 save_gar_version 
------------------
               78
(1 row)

CALL
CALL
 set_gar_dt_create 
-------------------
                78
(1 row)

CALL
gar_fias.as_apartment_type: 13
gar_fias.as_addr_obj_type: 421
gar_fias.as_room_type: 3
gar_fias.as_operation_type: 27
gar_fias.as_param_type: 20
gar_fias.as_house_type: 14
gar_fias.as_add_house_type: 4
gar_fias.as_object_level: 17
gar_fias.as_norm_docs_types: 25
gar_fias.as_norm_docs_kinds: 4
gar_fias.as_reestr_objects: 322584
gar_fias.as_addr_obj: 7538
gar_fias.as_houses: 410276
gar_fias.as_steads: 46805
gar_fias.as_apartments: 106817
gar_fias.as_rooms: 905
gar_fias.as_carplaces: 385
gar_fias.as_adm_hierarchy: 365316
gar_fias.as_mun_hierarchy: 334104
gar_fias.as_change_history: 1270442
gar_fias.as_addr_obj_division: 7
gar_fias.as_addr_obj_params: 113718
gar_fias.as_steads_params: 549569
gar_fias.as_houses_params: 2350767
gar_fias.as_apartments_params: 337830
gar_fias.as_rooms_params: 918
gar_fias.as_carplaces_params: 58
gar_fias.as_normative_docs: 52577
